types/src/transaction/mod.rs

```diff
@@ -641,7 +641,7 @@ impl SignedTransaction {
     }
 
     /// Returns the hash when the transaction is commited onchain.
-    pub fn committed_hash(self) -> HashValue {
+    pub fn lookup_hash(self) -> HashValue {
         Transaction::UserTransaction(self).hash()
```

Comment:
If you're going to rename this, I'd do `txn_hash` or `hash`, because `lookup_hash` makes it seem like you're making an outgoing network or DB to do it.